origin.spec

```diff
@@ -46,16 +46,16 @@
 %endif
 %{!?make_redistributable: %global make_redistributable %{need_redistributable_set}}
 
-%if "%{dist}" == ".el7aos"
+%if 0%{?epel}
 %global package_name openshift
 %global product_name OpenShift
 %else
-%global package_name openshift
-%global product_name OpenShift
+%global package_name okd
```

Comment:
aren't we breaking OCP here?

Reply:
hm, on epel it'll still be openshift. See the lines above

Reply:
This did actually break things in OCP. Should be fixed by the `.openshift` dist tag now

Reply:
Let's just use package_name openshift for everything.  Do we *need* to call it OKD?

Reply:
Oh, I see.  Yeah, let's call the variable `ocp_build`, keep the package name the same (no reason not to), and change product name based on ocp_build